src/utils/read-fiddle.ts

```diff
@@ -39,9 +38,8 @@ export async function readFiddle(folder: string): Promise<EditorValues> {
     console.warn(`readFiddle(): "${folder}" does not exist`);
   } else {
     for (const file of fs.readdirSync(folder)) {
-      const key = FILENAME_KEYS[file];
-      if (key) {
-        ret[key] = tryRead(file);
+      if (Object.values(DefaultEditorId).includes(file as DefaultEditorId)) {
+        ret[file] = tryRead(file);
```

Comment:
This doesn't seem to support loading custom editors from saved fiddles?

Reply:
ooh you're right, looks like i spaced that! i'll do the same permissions dance for non-default editors